online-dashboard-api/app/Models/User.php

```diff
@@ -45,4 +50,25 @@ protected function casts(): array
             'password' => 'hashed',
         ];
     }
+
+    public function passwordResetTokens(): HasMany
+    {
+        return $this->hasMany(ForgetPasswordTokens::class);
+    }
+
+    public function initiatePasswordReset(): void
+    {
+        $expiry = Carbon::now()->addMinutes(60);
```

Comment:
Make expiry count to 5 mins and get the value from config tooo

Reply:
done